WPFHospitalEditor/Controller/DoctorServerController.cs

```diff
@@ -20,9 +20,19 @@ public Doctor GetDoctorById(int doctorId)
             return doctorServerService.GetDoctorById(doctorId);
         }
 
+        public IEnumerable<DoctorDto> GetFilteredDoctors(string name)
+        {
+            return doctorServerService.GetFilteredDoctors(name);
+        }
+
```

Comment:
Name FilteredDoctor is confusing. It's better name it something like SearchDoctors